gateway-ha/src/test/java/io/trino/gateway/ha/TestGatewayHaMultipleBackend.java

```diff
@@ -62,6 +64,7 @@ final class TestGatewayHaMultipleBackend
 
     private TrinoContainer adhocTrino;
     private TrinoContainer scheduledTrino;
+    private final PostgreSQLContainer postgresql = new PostgreSQLContainer("postgres:16");
```

Comment:
There are many duplicated `postgres:16`. We should define a constant somewhere in a follow-up.